src/Bridges/lazybridgeoptimizer.jl

```diff
@@ -99,6 +99,11 @@ end
 function is_bridged(b::LazyBridgeOptimizer, F::Type{<:MOI.AbstractFunction}, S::Type{<:MOI.AbstractSet})
     !MOI.supports_constraint(b.model, F, S)
 end
+# Same as supports_constraint but do not trigger `update_constraint!`. This is
```

Comment:
Where is the implementation of `supports_constraint` for `LazyBridgeOptimizer`?

Reply:
in `bridgeoptimizer.jl`